app/models/riiif/image.rb

```diff
@@ -51,7 +51,10 @@ def render(args)
     def info
       @info ||= begin
                   result = info_service.call(id, file)
-                  ImageInformation.new(width: result[:width], height: result[:height])
+                  ImageInformation.new(
```

Comment:
I'd prefer we make `ImageInformation` have a dimensions attribute and add the format attribute.  Is there a reason we can't do that?

Reply:
Yeah, I think I was confused by the weird override of the `initialize` method.  Updated now though.